Tasks/FuncToolsInstallerV0/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-tool-lib": "0.12.0",
+    "func-tools-common": "file:../../_build/Tasks/Common/func-tools-common-1.0.0.tgz"
+  }
+}
```

Comment:
Add in other meta-data to the package.json?

Reply:
not doing as discussed.